### PR TITLE
fix: do not put a momentjs instance in the date field of a bill

### PR DESCRIPTION
**Pull request by doubleface**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -163,5 +163,6 @@ const getPdf = function (bill) {
 }
 
 function getFileName (date) {
+  date = moment(date)
```
Comment (briced) [inline index.js]: Could you declare a new variable here? Immutability FTW o//

Comment (briced) [inline index.js]: Fixed it myself.

**Comment by briced** [comment]
Version 1.0.6, tagged and deployed to https://github.com/briced/cozy-konnector-v3-ameli/tree/build

File: index.js (hunk 1 above)
Comment (doubleface) [inline index.js]: Thanks @briced. I am on holiday and just sent the pullrequest before it...

**Pull request merged**